### DISCUSSION TITLE
Sorry I accident delete namespace , when I get ns it's alway show Terminating

### DISCUSSION THREAD

**Discussion by Re3v3s** [argoproj/argo-cd#15586] [Q&A]
First I apologize with my bad English. So when I describe my namespace it's show like this
 
```
NamespaceContentRemaining                    True    Tue, 19 Sep 2023 22:26:19 +0700  SomeResourcesRemain   Some resources are remaining: applications.argoproj.io has 1 resource instances
  NamespaceFinalizersRemaining                 True    Tue, 19 Sep 2023 22:26:19 +0700  SomeFinalizersRemain  Some content in the namespace has finalizers remaining: resources-finalizer.argocd.argoproj.io/foreground in 1 resource instances
```

Someone told me to edit namespace and remove about finalize
  finalizers:
  - kubernetes

So when I edit and save , and I check file something that've deleted ,it's still there , I don't know why

**Comment by jannfis** [comment]
From the events above, it seems that there's a finalizer remaining on an `Application` resource.

You need to edit or patch that Application to remove the finalizer from there.

**Reply by Re3v3s** [reply]
Thank you , Sir , Finally I can deleted it.
